src/Symfony/Component/VarExporter/README.md

```diff
@@ -0,0 +1,33 @@
+VarExporter Component
+=====================
+
+The VarExporter component allows exporting any serializable PHP data structure to
+plain PHP code. While doing so, it preserves all the semantics associated to the
+serialization mechanism of PHP (`__wakeup`, `__sleep`, `Serializable` esp.)
+
+The reason to use this component *vs* `serialize()` or
+[igbinary](https://github.com/igbinary/igbinary) is performance: thanks to
+OPcache, the resulting code is significantly faster and more memory efficient
+than using `unserialize()` or `igbinary_unserialize()`.
+
+Unlike `var_export()`, this works on any serializable PHP value.
+
+It also provides a few improvements over `var_export()`/`serialize()`:
+ * the output is PSR-2 compatible
```

Comment:
missing empty line before the list, and missing `;` at the end of each line but the last.